apps/publish/content/common.py

```diff
@@ -895,8 +895,10 @@ def _publish_associated_items(self, original, updates=None):
                         sync_associated_item_changes(associated_item, associated_item_updates)
                         continue
 
-                    if association_updates.get("state") not in PUBLISH_STATES:
-                        # There's an update to the published associated item
+                    if (
+                        app.config.get("ENABLE_ASSOCIATED_ITEMS_UPDATE")
+                        or association_updates.get("state") not in PUBLISH_STATES
```

Comment:
I wouldn't work with a new config, I think it should be based on changes in the `association_updates` values,
imo that should be standard behaviour.